soc/arm/ambiq/Kconfig.defconfig

```diff
@@ -3,3 +3,9 @@
 # Copyright (c) 2023 Antmicro <www.antmicro.com>
 
 source "soc/arm/ambiq/*/Kconfig.defconfig.series"
+
+config CORTEX_M_SYSTICK
+	default n if AMBIQ_STIMER_TIMER
```

Comment:
is it default y?

Reply:
It is now - I've added `select CPU_CORTEX_M_HAS_SYSTICK`, to Kconfig.series (which enables `CONFIG_CORTEX_M_SYSTICK`): https://github.com/zephyrproject-rtos/zephyr/blob/a0858dcc312eb6115f9f7e713c6d7688524fc5bd/soc/arm/ambiq/apollo4x/Kconfig.series#L12

Reply:
this is indicating SoC has systick, but not enabling it

Reply:
i removed `select CPU_CORTEX_M_HAS_SYSTICK`.